ext/pdo_firebird/php_pdo_firebird_int.h

```diff
@@ -136,10 +134,17 @@ extern void php_firebird_set_error(pdo_dbh_t *dbh, pdo_stmt_t *stmt, const char
 #define php_firebird_error_with_info(d,e,el,m,ml) php_firebird_set_error(d, NULL, e, el, m, ml)
 #define php_firebird_error_stmt_with_info(s,e,el,m,ml) php_firebird_set_error(s->dbh, s, e, el, m, ml)
 
+extern bool php_firebird_commit_transaction(pdo_dbh_t *dbh, bool retain);
+
 enum {
 	PDO_FB_ATTR_DATE_FORMAT = PDO_ATTR_DRIVER_SPECIFIC,
 	PDO_FB_ATTR_TIME_FORMAT,
 	PDO_FB_ATTR_TIMESTAMP_FORMAT,
 };
 
+enum {
+	FB_TXN_RELEASE,
+	FB_TXN_RETAIN
+};
```

Comment:
Requires a typedef and this type to be passed to ``php_firebird_commit_transaction`` instead of ``bool``, or am I misunderstanding something here?

Reply:
I thought it would be difficult to understand if it was just a bool value, so I made it a constant.

```
php_firebird_commit_transaction(dbh, true) // retain
php_firebird_commit_transaction(dbh, false) // not retain
```

Maybe I'm overthinking it a bit, and maybe just passing a bool is fine...

Reply:
I will try to stop using constants (Enums).

Reply:
You can write:
```
php_firebird_commit_transaction(dbh, /* retain */ retain_var_or_const);
```
To imitate named arguments

Reply:
It's a good idea! Thank you!